src/styles/blocks/cal.scss

```diff
@@ -0,0 +1,46 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 7 * $cell-size;
+  border: 1px solid #000;
+
+  $days:
+    "mon" 0,
+    "tue" 1,
+    "wed" 2,
+    "thu" 3,
+    "fri" 4,
+    "sat" 5,
+    "sun" 6;
+
+  @each $day, $date in $days {
+    &--start-day--#{$day}:first-child {
+      margin-left: $cell-size * $date + $cell-gap * $date;
+    }
+  }
+
+  &__cell {
+    display: flex;
+    height: $cell-size;
+    width: $cell-size;
+    box-sizing: border-box;
+    border: 1px solid #000;
+    justify-content: center;
+    align-items: center;
```

Comment:
insert ` background-color` here 
and `transition` property here not in `hover` state

Reply:
I need to leave another background color for hover as well since it needs to change the color on hover.